File: DorotOlam/DorotOlam/src/app/memorial/models/record/record.ts
import { Family } from '../family/family';
import { Legacy } from '../legacy/legacy';
import { LifeStory } from '../lifeStory/life-story';
import { PersonalInformation } from '../personalInformation/personal-information';

export class Record {
    personalInformation:PersonalInformation;
    lifeStory:LifeStory;
    family:Family;
    legacy:Legacy;
}

File: DorotOlam/Models/Record.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DorotOlam.Models
{
    public class Record
    {
      public   PersonalInformation PersonalInformation { get; set; }
        public Legacy Legacy { get; set; }
        public LifeStory LifeStory { get; set; }

        public Family Family { get; set; }
    }
}
